### ISSUE TITLE
验证集百度云链接失效

### ISSUE THREAD

**Issue by jasonlbx13** [chenjun2hao/CenterFace.pytorch#29]
感谢您的分享！
验证集标注文件地址（下面这行）对应的百度云链接已失效
download the validation set of WIDER_FACE password: y4wg

方便再更新一下吗？谢谢！

**Comment by chenjun2hao** [comment]
i package the all data in one zip file. 链接: https://pan.baidu.com/s/1wfENmkz-ID4RiP-gbZDWFA 提取码: m53d
